modules/reference/pages/rpk/rpk-x-options.adoc

```diff
@@ -16,20 +16,136 @@ Every `-X` option can be translated into an environment variable by prefixing wi
 
 [TIP]
 ====
+* For persistent configuration across commands and sessions, we recommend using xref:get-started:config-rpk-profile.adoc[rpk profiles] instead of environment variables or `-X` flags.
 * `rpk` supports command-line (tab) completion for `-X` flag keys.
 * Each `rpk` command's `-help` text prints information specific to the command. To view a description of `-X` options, run `rpk -X list` to list supported options, or run `rpk -X help` to get details about supported options.
 ====
 
+== Configuration priority
+
+`rpk` resolves configuration values in the following priority order (highest to lowest):
+
+1. **Command-line flags** (including `-X` options): Highest priority, applies to current command only
+2. **Environment variables**: Medium priority, `RPK_*` env variables lasts for shell session
+3. **rpk profile** (`rpk.yaml`): Lower priority, persistent across sessions (recommended)
+4. **Redpanda configuration** (`redpanda.yaml` rpk section): Lowest priority, system-wide defaults
```

Comment:
Is this what's referred to in the [first line](https://deploy-preview-1306--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-x-options/) of this doc (`Use rpk -X flag to override any rpk-specific configuration option.`)? If so, I think it would be good to make that connection for the reader, perhaps by expanding the first line a bit, e.g.:
> ...rpk-specific configuration option set by default in `redpanda.yaml`

(something along those lines)